defaults/main.yml

```diff
@@ -41,6 +45,7 @@ sap_hana_deployment_hana_userid:
 sap_hana_deployment_hana_groupid:
 sap_hana_deployment_hana_db_system_password:
 sap_hana_deployment_system_restart: 'n'
+sap_hana_deployment_create_initial_tenant:
```

Comment:
I can't see any reference in the README to this variable, can you please add this?